control-plane-agent/internal/logic/actions/action-all-multipoint-groups-apply-proxy-star-interconnect.go

```diff
@@ -0,0 +1,356 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+package actions
+
+import (
+	"context"
+	"time"
+
+	"control-plane-agent/internal/event"
+	"control-plane-agent/internal/model"
+	"control-plane-agent/internal/registry"
+
+	"github.com/sirupsen/logrus"
+)
+
+type Action_AllMultipointGroupsApplyProxyStarInterconnect struct{}
+
+func (a *Action_AllMultipointGroupsApplyProxyStarInterconnect) ValidateModifier(modifier string) error {
+	return nil
+}
+
+func (a *Action_AllMultipointGroupsApplyProxyStarInterconnect) Perform(ctx context.Context, modifier string, param event.Params) (context.Context, bool, error) {
+
+	groups, err := registry.MultipointGroupRegistry.List(ctx, nil, false, false)
+	if err != nil {
+		return ctx, false, nil
+	}
+	if len(groups) == 0 {
+		return ctx, true, nil
+	}
+	conns, err := registry.ConnRegistry.List(ctx, nil, false, true)
+	if err != nil {
+		return ctx, false, nil
+	}
+	connsNum := len(conns)
+	if connsNum == 0 {
+		return ctx, true, nil
+	}
+	proxies, err := registry.MediaProxyRegistry.List(ctx, nil, false, true)
+	if err != nil {
+		return ctx, false, nil
+	}
+	proxiesNum := len(proxies)
+	if proxiesNum == 0 {
+		return ctx, true, nil
+	}
+	bridges, err := registry.BridgeRegistry.List(ctx, nil, false, true)
+	if err != nil {
+		return ctx, false, nil
+	}
+	bridgesNum := len(bridges)
```

Comment:
Shouldn't there be a check for bridgesNum?
```
	if bridgesNum == 0 {
		return ctx, true, nil
	}
```

Reply:
No, in this interconnect algorithm the number of bridges can be zero. Since it's a valid value, there is no reason to return an error.